order/buyer/issue.md

```diff
@@ -0,0 +1,133 @@
+---
+description: How to issue a new order as a buyer
+---
+
+# Issue a new order
+
+As buyer you can send either a new or [updated](reissue.md) purchase order to Tradecloud.
+
+## Send a new order to Tradecloud
+
+As a buyer you can send a new purchase order to Tradecloud.
+
+When the `/order-integration/order` API method returns HTTP status code 200, the order was successfully queued to be processed by Tradecloud. Processing takes usually less then a second, after which to order is available in the portal and is forwarded to the supplier ERP integration.
```

Comment:
... after which *the* order ...

Reply:
Fixed, and moved to new `##Response` section